### PR TITLE
chore: fix base-codefixes test

### PR DESCRIPTION
**Pull request by egorio**
Currently, test for base codefixes do nothing and snapshots contain original (not modified) files contents... this is why they pass. 

This PR is fixing those tests.

### CODE REVIEW

File: packages/migrate/src/glint-utils.ts [hunk 1]
```diff
@@ -1,41 +1,10 @@
...
-// The list of extensions that we expect to be handled by Glint{Fix,Check} plugins. Note that
-// in any ember/glimmer project, we'll use the glint *service* for all files. This list is only
-// indicating which extensions are handled by the plugins
-export const GLINT_EXTENSIONS = ['.gts', '.hbs'];
-
-// The list of dependencies we look for to determine if we're in a glint project. If we find one
-// of these, we use glint. Otherwise, we use the regular Rehearsal service
-export const GLINT_PROJECT_FILES = ['ember-source', '@glimmer/component', '@glimmerx/component'];
-
-export async function shouldUseGlint(basePath: string): Promise<boolean> {
```
Comment (egorio) [inline packages/migrate/src/glint-utils.ts:21]: Clean up

Comment (wondersloth) [reply packages/migrate/src/glint-utils.ts:21]: Why can we remove this?

Comment (egorio) [reply packages/migrate/src/glint-utils.ts:21]: It's not used.

Comment (egorio) [reply packages/migrate/src/glint-utils.ts:21]: We have same in @service

**Review by wondersloth — APPROVED** [review]

**Pull request merged**